docs/source/model_doc/marian.rst

```diff
@@ -99,16 +130,23 @@ For example:
      'CELTIC': ['ga', 'cy', 'br', 'gd', 'kw', 'gv']
     }
 
-Code to see available pretrained models:
 
-.. code-block:: python
 
-    from transformers.hf_api import HfApi
-    model_list = HfApi().model_list()
-    org = "Helsinki-NLP"
-    model_ids = [x.modelId for x in model_list if x.modelId.startswith(org)]
-    suffix = [x.split('/')[1] for x in model_ids]
-    multi_models = [f'{org}/{s}' for s in suffix if s != s.lower()]
+
+Example of translating english to many romance languages, using old-style 2 character language codes
+
+
+.. code-block::python
+
+    from transformers import MarianMTModel, MarianTokenizer
+     src_text = [ '>>fr<< this is a sentence in english that we want to translate to french', '>>pt<< This should go to portuguese', '>>es<< And this to Spanish']
```

Comment:
You got some bad indent here. It then made the rest of the code badly formatted, so you need to re-add the new lines at the right places.